launcher/ui/pages/instance/ModFolderPage.cpp

```diff
@@ -86,12 +88,30 @@ ModFolderPage::ModFolderPage(BaseInstance* inst, std::shared_ptr<ModFolderModel>
         ui->actionsToolbar->insertActionAfter(ui->actionAddItem, ui->actionUpdateItem);
         connect(ui->actionUpdateItem, &QAction::triggered, this, &ModFolderPage::updateMods);
 
+        ui->actionVisitItemPage->setToolTip(tr("Go to mod's home page"));
+        ui->actionsToolbar->insertActionAfter(ui->actionViewFolder, ui->actionVisitItemPage);
+        connect(ui->actionVisitItemPage, &QAction::triggered, this, &ModFolderPage::visitModPages);
+
         auto check_allow_update = [this] {
             return (!m_instance || !m_instance->isRunning()) && (ui->treeView->selectionModel()->hasSelection() || !m_model->empty());
         };
 
-        connect(ui->treeView->selectionModel(), &QItemSelectionModel::selectionChanged, this,
-                [this, check_allow_update] { ui->actionUpdateItem->setEnabled(check_allow_update()); });
+        connect(ui->treeView->selectionModel(), &QItemSelectionModel::selectionChanged, this, [this, check_allow_update] {
+            ui->actionUpdateItem->setEnabled(check_allow_update());
+
+            auto selection = m_filterModel->mapSelectionToSource(ui->treeView->selectionModel()->selection()).indexes();
+            auto mods_list = m_model->selectedMods(selection);
+            auto selected = std::count_if(mods_list.cbegin(), mods_list.cend(),
+                                          [](Mod* v) { return v->metadata() != nullptr || v->homeurl().size() != 0; });
+            if (selected <= 1) {
+                ui->actionVisitItemPage->setText(tr("Visit on mod's page"));
```

Comment:
I feel like this sounds better
```suggestion
                ui->actionVisitItemPage->setText(tr("Visit mod's page"));
```